tests/foreman/api/test_contentview.py

```diff
@@ -60,8 +62,50 @@
 REPEAT = 3
 
 
+@fixture(scope='class', autouse=True)
+def module_org(request):
+    request.cls.module_org = entities.Organization().create()
+
+
+@fixture(scope='class', autouse=True)
+def module_lce(request, module_org):
+    request.cls.module_lce = entities.LifecycleEnvironment(
+        organization=request.cls.module_org).create()
+
+
+@fixture(scope='class', autouse=True)
+def module_product(request, module_org):
+    request.cls.module_product = entities.Product(
+        organization=request.cls.module_org).create()
+
+
+@pytest.mark.usefixtures('module_org', 'module_lce', 'module_product')
```

Comment:
Can we keep all the fixtures in conftest file to avoid the bulkiness of test case file?

Reply:
We can even keep in Single file (not in conftest to avoid getting it bulky) it can be used across other files. e.g. https://gist.github.com/peterhurford/09f7dcda0ab04b95c026c60fa49c2a68 mentioned here

Reply:
@devendra104, That's a really good idea but as I said there are higher chances of collision and failures if we do CRUD to those entity objects. These objects are mutable and hence having them as test modules is OK and can be handled. But for the entire test bed it's quite uncontrollable.

Still, there could be few entities like org and loc can be used for the entire testbed as fixture (except org and loc itself) which are less likely to operate on.

Reply:
@devendra104, @omkarkhatavkar  That's a really good idea but as I said there are higher chances of collision and failures if we do CRUD to those entity objects. These objects are mutable and hence having them as test modules is OK and can be handled. But for the entire test bed it's quite uncontrollable.

Still, there could be few entities like org and loc can be used for the entire testbed as fixture (except org and loc itself) which are less likely to operate on.

Reply:
@omkarkhatavkar @devendra104, All the fixtures are now moved to a special library under robottelo/api/entity_fixtures.py and those all has been imported to conftest. The approach works great.

Reply:
@jyejare Awesome Work!